SPECS/ypbind/ypbind.spec

```diff
@@ -80,17 +79,17 @@ export LDFLAGS="$LDFLAGS -pie -Wl,-z,relro,-z,now"
 %install
 %make_install
 
-mkdir -p $RPM_BUILD_ROOT%{_localstatedir}/yp/binding
-mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/dhcp/dhclient.d
-mkdir -p $RPM_BUILD_ROOT%{_unitdir}
-mkdir -p $RPM_BUILD_ROOT%{_libexecdir}
-install -m 644 etc/yp.conf $RPM_BUILD_ROOT%{_sysconfdir}/yp.conf
-install -m 755 %{SOURCE2} $RPM_BUILD_ROOT%{_sysconfdir}/dhcp/dhclient.d/nis.sh
-install -m 644 %{SOURCE3} $RPM_BUILD_ROOT%{_unitdir}/ypbind.service
-install -m 755 %{SOURCE4} $RPM_BUILD_ROOT%{_libexecdir}/ypbind-pre-setdomain
-install -m 755 %{SOURCE5} $RPM_BUILD_ROOT%{_libexecdir}/ypbind-post-waitbind
+mkdir -p %{buildroot}%{_localstatedir}/yp/binding
+mkdir -p %{buildroot}%{_sysconfdir}/dhcp/dhclient.d
+mkdir -p %{buildroot}%{_unitdir}
+mkdir -p %{buildroot}%{_libexecdir}
+install -m 644 etc/yp.conf %{buildroot}%{_sysconfdir}/yp.conf
+install -m 755 %{SOURCE2} %{buildroot}%{_sysconfdir}/dhcp/dhclient.d/nis.sh
+install -m 644 %{SOURCE3} %{buildroot}%{_unitdir}/ypbind.service
+install -m 755 %{SOURCE4} %{buildroot}%{_libexecdir}/ypbind-pre-setdomain
+install -m 755 %{SOURCE5} %{buildroot}%{_libexecdir}/ypbind-post-waitbind
 
-%{find_lang} %{name}
+%find_lang %{name}
```

Comment:
That's change is a nice catch. Did you do it manually or did the linter find it? I'm assuming it was a manual change but if not, it looks like we might get more out of the linter than I anticipated.

Reply:
The linter finds it.